pom.xml

```diff
@@ -58,7 +58,7 @@
         <attachmentsVersion>2.8.0</attachmentsVersion>
         <authenticationVersion>1.0.0</authenticationVersion>
         <authenticationuiVersion>1.1.0-SNAPSHOT</authenticationuiVersion>
-        <bedmanagementVersion>6.0.0</bedmanagementVersion>
+        <bedmanagementVersion>6.1.0-SNAPSHOT</bedmanagementVersion>
```

Comment:
Looks good!  The automatic version plugin will auotmatically update this to 6.1.0 when that version is released.